### ISSUE TITLE
Doesn't work with  build_runner 2.0.4

### ISSUE THREAD

**Issue by kristijorgji** [vanlooverenkoen/kiwi#71]
`Dart SDK version: 2.13.1 (stable) (Fri May 21 12:45:36 2021 +0200) on "macos_x64"`
`Flutter 2.2.2 • channel stable • https://github.com/flutter/flutter.git
Framework • revision d79295af24 (11 days ago) • 2021-06-11 08:56:01 -0700
Engine • revision 91c9fc8fe0
Tools • Dart 2.13.3`

Using build_runner version: "2.0.4"

build.yml content:
```
targets:
  $default:
    builders:
      kiwi:

```

Running same script as before upgrade to new version is giving:

`flutter packages pub run build_runner build`
```
If you believe you have gotten this message in error, especially if using a new
feature, you may need to run `pub run build_runner clean` and then rebuild.

Invalid argument(s): line 4, column 7 of build.yaml: Unsupported value for "builders". Must not be null
  ╷
4 │       kiwi:
  │       ^^^^^
```

**Comment by zairahjy** [comment]
Same thing happens to me. Please help us.

**Comment by kristijorgji** [comment]
> Same thing happens to me. Please help us.

I solved it friend, solution was to delete `build.yaml` then running builder works.
The kiw_generator documentation that says to create build.yaml with config above is wrong for new version of build_runner

**Comment by zairahjy** [comment]
Yeah, it's working without the yaml file. Please update the docs.

**Comment by vanlooverenkoen** [comment]
Will be fixed in #72

**Comment by vanlooverenkoen** [comment]
Fixed in 3.0.1

**Comment by MaldonadoAndres** [comment]
still not working with build_runner: 2.1.2
